perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Add license and document what the user needs to specify in this file (e.g. what all are required parameters, that user can't specify the configuration name, what uniquely defines an experiment etc)